src/main/scala/com/tuplejump/kafka/connect/cassandra/TaskConfig.scala

```diff
@@ -123,6 +125,9 @@ object TaskConfig {
   final val SinkConsistency: Key = "cassandra.sink.consistency"
   final val DefaultSinkConsistency = ConsistencyLevel.LOCAL_QUORUM
 
+  final val FieldMapping: Key = "cassandra.sink.field.mapping"
+  final val DefaultFieldMapping = Map.empty[String, Any]
```

Comment:
I don't see how this could be anything other than Map[String,String] from the overarching config in the api so

         final val DefaultFieldMapping = Map.empty[String, String]